politeiawww/backend_proposal_test.go

```diff
@@ -459,7 +460,8 @@ func TestCensoredProposal(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	censorProposal(b, npr.CensorshipRecord.Token, t, user, id)
+	csrMessage := "unappropriated content"
+	censorProposal(b, npr.CensorshipRecord.Token, csrMessage, t, user, id)
 	pdr := getProposalDetails(b, npr.CensorshipRecord.Token, t)
 	verifyProposalDetails(np, pdr.Proposal, t)
```

Comment:
Missing check?

Reply:
This isn't possible right now because the message is only stored on d and the inventory isn't updating the record's metadata as reported here https://github.com/decred/politeia/issues/396. Would be better to add the check once the fix for the inventory gets merged in.

Reply:
A verification of it isn't possible right now because the message is only stored on d and the inventory isn't properly updating the proposal metadata as reported here https://github.com/decred/politeia/issues/396.